tests/js/spec/components/avatar.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import Avatar from 'app/components/avatar';
```

Comment:
Awesome – tests!

However, I'd like to see us use [Enzyme](https://github.com/airbnb/enzyme) as much as possible. It's seriously easier to write tests with. There's even an internal conversation on the React core team about deprecating TestUtils in favor of Enzyme (but currently unresolved).

https://github.com/reactjs/core-notes/blob/master/2016-05/may-05.md#testutils-and-enzyme
https://github.com/facebook/react/pull/6523